src/wp-content/themes/twentytwentythree/style.css

```diff
@@ -0,0 +1,16 @@
+/*
+Theme Name: Twenty Twenty-Three
+Theme URI: https://github.com/WordPress/twentytwentythree
+Author: wordpressdotorg
+Author URI: https://wordpress.org
+Description:
```

Comment:
Missing description.

Reply:
I've created an issue in the TT3 repo for this here: https://github.com/WordPress/twentytwentythree/issues/177

Is this needed for Beta 1?

Reply:
I've added a description so this is included for Beta 1 (https://github.com/WordPress/wordpress-develop/pull/3280/commits/ea8c44f138eda367186a7e6ab721d92d4b9fb2b3).